code/script.js

```diff
@@ -1,53 +1,209 @@
+  <button name="plant-button" class="medium-button" id="mediumButton" value="medium" type="button">medium</button>
+  <button name="plant-button" class="large-button" id="largeButton" value="large" type="button">large</button>`;
+
+  const smallButton = document.getElementById("smallButton");
+  const mediumButton = document.getElementById("mediumButton");
+  const largeButton = document.getElementById("largeButton");
+
+  smallButton.addEventListener("click", chosedPlantSize);
+  mediumButton.addEventListener("click", chosedPlantSize);
+  largeButton.addEventListener("click", chosedPlantSize);
+};
+
+const chosedPlantSize = (event) => {
+  const selectedSize = event.target.value;
+  userMessage(`I choose ${selectedSize}`);
+  nameForm.innerHTML = "";
+  setTimeout(handlePlantSize, 1000, selectedSize);
+};
 
-// Here we invoke the first function to get the chatbot to ask the first question when
-// the website is loaded. Normally we invoke functions like this: greeting()
-// To add a little delay to it, we can wrap it in a setTimeout (a built in JavaScript function):
-// and pass along two arguments:
-// 1.) the function we want to delay, and 2.) the delay in milliseconds 
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greetUser, 1000)
+const handlePlantSize = (selectedSize) => {
+  if (selectedSize === "small") {
+    botMessage(`You want a ${selectedSize}, what plant do you want?`);
+    nameForm.innerHTML = `<select id="smallPlants" name="small-plants">
+  <option disabled selected>Choose your plant</option>
+  <option>Spiderplant</option>
+  <option>Mini succulent</option>
+  <option>Air plant</option>`;
+    const smallPlants = document.getElementById("smallPlants");
+    smallPlants.addEventListener("change", selectedPlantType);
+  } else if (selectedSize === "medium") {
+    botMessage(`You want a ${selectedSize}, what plant do you want?`);
+    nameForm.innerHTML = `<select id="mediumPlants"  name="medium-plants">
+  <option disabled selected>Choose your plant</option>
+  <option>Money Tree</option>
+  <option>Schefflera</option>
+  <option>Peach Lily</option>`;
+    const mediumPlants = document.getElementById("mediumPlants");
+    mediumPlants.addEventListener("change", selectedPlantType);
+  } else {
+    botMessage(`You want a ${selectedSize}, what plant do you want?`);
+    nameForm.innerHTML = `<select id="largePlants" name="large-plants">
+  <option disabled selected>Choose your plant</option>
+  <option>Monstera</option>
+  <option>Bird of Paradise</option>
+  <option>Olive Tree</option>`;
+    const largePlants = document.getElementById("largePlants");
+    largePlants.addEventListener("change", selectedPlantType);
+  }
+};
+
+const selectedPlantType = (event) => {
+  const selectedType = event.target.value;
+  userMessage(`I choose ${selectedType}`);
+  nameForm.innerHTML = "";
+  setTimeout(handlePlantType, 1000, selectedType);
```

Comment:
This syntax is new to me, but seems to be working as good as:
`setTimeout(() => handlePlantType(selectedType), 1000);`